tests/test_overdrive.py

```diff
@@ -146,7 +146,7 @@ def test_place_hold_looks_up_notification_address(self):
         # And when we placed it on hold, we passed in foo@bar.com
         # as the email address -- not notifications@example.com.
         hold_request = api.requests[-1]
-        body = hold_request[4]
+        body = hold_request[3][1]
```

Comment:
The core `MockOverdriveAPI` doesn't put the request body in a special slot in the array it appends to `request`--it just puts all positional arguments to the "make a request" method into the fourth position of the array. The body happens to be the second positional argument.